modules/internal/ChapelArray.chpl

```diff
@@ -438,6 +438,7 @@ module ChapelArray {
   // the low bound to use for array literals
   pragma "no doc"
   config param arrayLiteralLowBound = defaultLowBound;
+  config param capturedIteratorLowBound = defaultLowBound;
```

Comment:
`pragma "no doc"` this?

Reply:
Oops, yes, thanks!

Reply:
Addressed in https://github.com/chapel-lang/chapel/pull/14800/commits/98f255a83be713c8418e350551910f4cc0e5769e